src/components/PlatformSetting/index.tsx

```diff
@@ -36,15 +49,88 @@ class PlatformSetting extends React.Component<Props> {
       }
       const { enableCollection, loginType, velaAddress } = values;
       const param = { enableCollection, loginType, velaAddress };
-      updateSystemInfo(param).then((res) => {
-        if (res) {
-          Message.success(i18n.t('Update the platform configuration success'));
-          this.props.syncPlatformSetting();
-        }
-      });
+      updateSystemInfo(param)
+        .then((res) => {
+          if (res) {
+            Message.success(i18n.t('Update the platform configuration success'));
+            this.props.syncPlatformSetting();
+          }
+        })
+        .catch((err) => {
+          const businessGuideCode = err?.BusinessCode || 0;
+          this.setState(
+            {
+              businessGuideCode: businessGuideCode,
+            },
+            () => {
+              if (this.state.businessGuideCode === 12011) {
+                this.renderDexGuideDialog();
+              } else if (this.state.businessGuideCode === 14010) {
+                this.renderUserGuideDialog();
+              }
+            },
+          );
+        });
+    });
+  };
+
+  renderUserGuideDialog = () => {
+    return Dialog.alert({
+      title: i18n.t('Email address empty'),
```

Comment:
The email address of administrator is empty.

Reply:
done